pkgs/games/factorio/default.nix

```diff
@@ -17,7 +17,7 @@ let
   variants = {
     x64 = {
       url = "https://www.factorio.com/get-download/${version}/alpha/linux64";
-      sha256 = "1xpzrx3q678519qgjl92fxn3qv55hd188x9jp6dcfk2ljhi1gmqk";
+      sha256 = "1dc53swnlbz23czw8v3ldl4ayp7118qyja4vh1z5dsd871swz9qg";
     };
 
     x32 = {
```

Comment:
Absolutely. It didn't occur to me at the time, since I was just scratching my own itch.

That would be a separate derivation, though it could probably share the config stuff. Note that, although it's still unfree, http://www.factorio.com/download-headless/stable allows you to download the headless version *without* having a username/password—that's certainly deliberate.